main.go

```diff
@@ -41,6 +41,13 @@ func getMetrics(zkOpts zkOptions) map[string]string {
 		if err != nil {
 			log.Printf("warning: cannot connect to %s: %v", h.Unresolved, err)
 			metrics[fmt.Sprintf("zk_up{%s}", hostLabel)] = "0"
+			newTcp, _ := net.ResolveTCPAddr("tcp", h.Unresolved)
+			if newTcp.IP.String() != h.TCPAddr.IP.String() {
+				log.Printf("INFO: resolve %s to new ip: %s ( old ip was: %s )", h.Unresolved, newTcp, h.TCPAddr)
```

Comment:
Can we keep log level naming consistent and replace `warning` to `WARNING` then in other log messages ?
I think `newTcp` should be `newTcp.IP` here.